pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -52,6 +52,7 @@ const _ = cockpit.gettext;
 
 const URL_SOURCE = 'url';
 const LOCAL_INSTALL_MEDIA_SOURCE = 'file';
+const EXISTING_DISK_IMAGE_SOURCE = 'disk_image';
```

Comment:
This commit fixes #11163 and the corresponding downstream bug https://bugzilla.redhat.com/show_bug.cgi?id=1666825 . Can you please reference them in the commit log, to make them easier to find?

Reply:
Done